src/mon/FSCommands.cc

```diff
@@ -448,6 +448,18 @@ class SetHandler : public FileSystemCommandHandler
       {
         fs.get_mds_map().set_max_filesize(n);
       });
+    } else if (var == "snap_rstat") {
+      bool snap_rstat = true;
+      int r = parse_bool(val, &snap_rstat, ss);
+      if (r != 0) {
+        return r;
+      }
+      fsmap.modify_filesystem(
+          fs->fscid,
```

Comment:
There is no variable named `fs` in this code region and `fscid` is private, therefore you need to call `get_fscid()` instead.

```suggestion
          fsp->get_fscid(),
```